src/Symfony/Component/CssSelector/Parser/Token.php

```diff
@@ -82,20 +82,12 @@ public function getPosition()
         return $this->position;
     }
 
-    /**
-     * @return bool
-     */
-    public function isFileEnd()
+    public function isFileEnd(): bool
     {
         return self::TYPE_FILE_END === $this->type;
     }
 
-    /**
-     * @param array $values
-     *
-     * @return bool
-     */
-    public function isDelimiter(array $values = array())
+    public function isDelimiter(array $values = array()): bool
```

Comment:
Sorry I deleted my original comment after noticing a thread saying just that, added to here https://github.com/symfony/symfony/pull/23262#discussion_r127008685 for reference